src/ManageContainer/Client/ManageToDb/Client.go

```diff
@@ -188,6 +190,23 @@ func getComputationStatus(path string) (pb_types.JobStatus, *pb_types.JobErrorIn
 	return pb_types.JobStatus_UNKNOWN, nil, errorAnyStatusFileNotFound
 }
 
+func (c Client) GetJobErrorInfo(jobUUID string) (*pb_types.JobErrorInfo, error) {
+	ls.Lock(jobUUID)
+	defer ls.Unlock(jobUUID)
+
+	path := fmt.Sprintf("%s/%s", resultDbPath, jobUUID)
+
+	_, errInfo, errStatus := getComputationStatus(path)
+	if errStatus != nil {
+		return nil, errStatus
+	}
+	if errInfo != nil {
+		return errInfo, nil
+	}
+
+	return &pb_types.JobErrorInfo{}, nil
```

Comment:
```suggestion
	return nil, nil
```

Reply:
Corrected with a [cf193dc](https://github.com/acompany-develop/QuickMPC/pull/100/commits/cf193dcd7535268822e062dcdf0a3fb1ecfbb700)

Reply:
Thank you!